### ISSUE TITLE
Request: A standalone vamp plugin graphical host

### ISSUE THREAD

**Issue by mx** [sonic-visualiser/sonic-visualiser#69]
This might be way out of scope for the project; if so, feel free to just close it.

I love Sonic Visualizer and the myriad of vamp plugins, but it can be a pain to use as part of a workflow of active real-time experimentation, rather than offline analysis. It would be so helpful if there was a simple vamp plugin container that could be given audio data and it would display its output in a GUI similar to one of Sonic Visualizer's panes. You could imagine someone then wrapping the plugin container in another container like JUCE so it could be used in a DAW.

**Comment by cannam** [comment]
I agree that that would be very nice! I don't have the time/funding to offer to work on it myself at the moment unfortunately.
